src/people/widgetViews/PhasePlannerView.tsx

```diff
@@ -59,6 +59,16 @@ const PhasePlannerView: React.FC = () => {
       console.log('Socket connected in Phase Planner');
     };
 
+    const refreshSingleTicket = async (ticketUuid: string) => {
+      try {
+        const ticket = await main.getTicketDetails(ticketUuid);
+
+        phaseTicketStore.updateTicket(ticket.uuid, ticket);
+      } catch (error) {
+        console.error('Error on refreshing ticket:', error);
+      }
+    };
+
     socket.onmessage = (event: MessageEvent) => {
```

Comment:
@MahtabBukhari could you add a log here of the initial websocket message as soon as its received to help with debugging?

Reply:
@humansinstitute https://github.com/stakwork/sphinx-tribes-frontend/pull/719